packages/block-library/src/image/edit.js

```diff
@@ -456,7 +457,14 @@ class ImageEdit extends Component {
 						label={ __( 'Alt Text (Alternative Text)' ) }
 						value={ alt }
 						onChange={ this.updateAlt }
-						help={ __( 'Alternative text describes your image to people who can’t see it. Add a short description with its key details.' ) }
+						help={
+							<Fragment>
+								<ExternalLink href={ __( 'https://www.w3.org/WAI/tutorials/images/decision-tree' ) }>
```

Comment:
Is it intended this link value be localized?

Reply:
We're planning to copy the W3C alt decision tree tutorial to the Handbook so it can be translated. But yes, can be non-translatable for now, will adjust.